app/models/user.rb

```diff
@@ -80,8 +80,8 @@ def student
   validates :homepage, format: { with: URL_PREFIX_PATTERN }, allow_blank: true
   validate :immutable_github_handle
 
-  accepts_nested_attributes_for :attendances, allow_destroy: true
-  accepts_nested_attributes_for :roles, allow_destroy: true
```

Comment:
quick question. what is the reason to remove the `allow_destroy: true` for roles?